modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/PeerDiscoveryDelay.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.sdk.domain.snapshot
+
+trait PeerDiscoveryDelay[F[_]] {
+
+  def waitForPeers: F[Unit]
```

Comment:
shouldn't it be parametrized by the number of peers to wait for?

Reply:
`checkPeersAttemptDelay`, `checkPeersMaxDelay`, `additionalDiscoveryDelay`, and `minPeersCount` have been added as parameters of the `make` and their values added to `AppConfig`.